Sources/TripKitUI/cards/TKUITimetableCard.swift

```diff
@@ -300,7 +300,9 @@ public class TKUITimetableCard : TKUITableCard {
       .take(1)
       .delay(.milliseconds(250), scheduler: MainScheduler.instance)
       .subscribe(onNext: { indexPath in
-        tableView.scrollToRow(at: indexPath, at: .top, animated: false)
+        if tableView.numberOfSections > indexPath.section, tableView.numberOfRows(inSection: indexPath.section) > indexPath.row {
```

Comment:
Looks fine. I just prefer if the other way around:

```suggestion
        if indexPath.section < tableView.numberOfSections, indexPath.row < tableView.numberOfRows(inSection: indexPath.section) {
```